packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -50,6 +50,27 @@
         /* Making the wrapper stack above the body */
         overscroll-behavior: none;
         z-index: 2;
+
+        .trade-notifications {
+            position: absolute;
+            width: 40rem;
+            max-width: calc(100% - 1.6rem);
+            z-index: 6;
```

Comment:
I'm curious Is there any reason to make it exactly 6? 😄

Reply:
@akmal-deriv yes, because this z-index is used be our existing pop-up notifications :) this is for consistency beside the fact that the new ones and the old ones can be displayed together :)